### PR TITLE
Ravinder | Development | Purchase order 1.2

### PR DESCRIPTION
**Pull request by ravinderwalkover**
Ravinder | Development | Purchase order 1.2

### CODE REVIEW

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 1]
```diff
@@ -484,7 +485,7 @@
                                                                 aria-autocomplete="none"></textarea>
                                                             <div>
                                                                 <div class="col-sm-6 pd">
-                                                                    <label class="sales-country-label">
+                                                                    <label class="salesCountryLabel">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:488]: Avoid camelCase in class name

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html [hunk 2]
```diff
@@ -1660,6 +1661,10 @@
     [selectedAccountUniqueName]="selectedVendorForDetails" (onShortcutPress)="toggleAccountAsidePane()"
     (addEvent)="addNewSidebarAccount($event)" (updateEvent)="updateSidebarAccount($event)"></generic-aside-menu-account>
 
+<aside-menu-product-service *ngIf="asideMenuStateForProductService === 'in'" [class]="asideMenuStateForProductService"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:1664]: Is there any use of `[class]="asideMenuStateForProductService"` as no class is mentioned in class property that will get applied on truthy value of `asideMenuStateForProductService`

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html [hunk 3]
```diff
@@ -106,7 +106,17 @@ <h4 class="clearfix bdrB">Purchase Order</h4>
...
                                         <div class="form-group clearfix">
                                             <input type="checkbox" class="hide-on-mobile" name="useCustomPONumber" [(ngModel)]="invoiceSettings.purchaseBillSettings.useCustomPONumber" />
-                                        </div>
+                                            <div class=" d-inline-flex custom-estimation" *ngIf="invoiceSettings.purchaseBillSettings.useCustomPONumber">
+                                                <div class="d-inline-flex pos-rel">
+                                                    <label>Initial No.</label>
+                                                    <input type="text" class="form-control text-right initialnumber" name="initialNumber" [(ngModel)]="invoiceSettings.purchaseBillSettings.initialPONumber">
+                                                </div>
+                                                <div class="d-inline-flex pos-rel">
+                                                    <label style="right:0px;">Prefix</label>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html:115]: Avoid inline styles

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.scss [hunk 4]
```diff
@@ -16,6 +16,24 @@
...
+        .custom-estimation label {
+            position: absolute;
+            top: -24px;
+            right: 15px;
+            font-size: 12px;
+            color: #666666;
+            text-align: right;
+        }
+        
+        .initialnumber {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.scss:33]: Do use dash-case syntax

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:1664]: in/out class gets applied on it.

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**